tvb_build/hbp_openshift_deployer/tvb_storage_backup.yml

```diff
@@ -0,0 +1,63 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: tvb-storage-backup
+  annotations:
+    description: "Template to create a cronjob that copies tvb-storage into a backup volume"
+    tags: "tvb, thevirtualbrain, hbp"
+
+objects:
+  - kind: "PersistentVolumeClaim"
+    apiVersion: v1
+    metadata:
+      name: storage-backup
+      annotations:
+        volume.beta.kubernetes.io/storage-class: managed-nfs-storage
+        volume.beta.kubernetes.io/storage-provisioner: fuseim.pri/ifs
+    spec:
+      accessModes:
+        - "ReadWriteOnce"
+      resources:
+        requests:
+          storage: "2Ti"
+
+  - kind: CronJob
+    apiVersion: batch/v1beta1
+    metadata:
+      name: tvb-postgres-backup
+    spec:
+      # ┌───────────── minute (0 - 59)
+      # │ ┌───────────── hour (0 - 23)
+      # │ │ ┌───────────── day of the month (1 - 31)
+      # │ │ │ ┌───────────── month (1 - 12)
+      # │ │ │ │ ┌───────────── day of the week (0 - 6) (Sunday to Saturday;
+      # │ │ │ │ │                                   7 is also Sunday on some systems)
+      # │ │ │ │ │
+      # │ │ │ │ │
+      # * * * * *
+      schedule: "0 0 * * 6"
+      concurrencyPolicy: "Forbid"
+      suspend: false
+      successfulJobsHistoryLimit: 7
+      failedJobsHistoryLimit: 7
+      jobTemplate:
+        spec:
+          template:
+            spec:
+              containers:
+                - name: tvb-postgres-backup
```

Comment:
Is this expected to do an rsync also on the DB disk? or maybe this name is just from copy+paste ?